[DIVD-2022-00053] Initiation of H2 case (#458)
DIVD-NL/web-csirt: _cases/2022/DIVD-2022-00053.md

---
layout: case

> title: DIVD-2022-00053 - H2 Web Console - CVE-2021-42392, CVE-2022-23221

Reviewer: ```suggestion
title: H2 Web Console - CVE-2021-42392, CVE-2022-23221
```

Reviewer: Onnodig, wordt uit de filenaam gehaald tegenwoordig.

> divd: DIVD-2022-00053

Reviewer: ```suggestion
```

Reviewer: Onnodig, wordt uit de filenaam gehaald tegenwoordig.

author: Martin van Wingerden
lead: Martin van Wingerden

> redirect_from:
> - /DIVD-2022-00053/

Reviewer: ```suggestion
```
Onnodig, wordt uit de filenaam gehaald tegenwoordig.

status: Open
excerpt: DIVD is researching vulnerable publicly exposed H2 Consoles
researchers:
- Martin van Wingerden

> cves: CVE-2021-42392, CVE-2022-23221

Reviewer: ```suggestion
cves: 
- CVE-2021-42392
- CVE-2022-23221
```
Komt netter op de site zo en dan worden ze ook ge hyperlinkt.

product: H2 Web Console
versions: 1.1.100-2.0.204
recommendation: "Update and do not expose the console to untrusted users."

> -workaround: n/a

Reviewer: ```suggestion
workaround: Do not expose the console to untrusted servers
```

start: 2022-09-09
end:
timeline:
- start: 2022-01-05
  end:
  event: "H2 reported about the vulnerability."
- start: 2022-09-09
  end:
  event:  DIVD starts scanning for vulnerabilities.
---

## Summary

On January 5 H2 published an advisory about CVE-2021-42392 and released a new version.

## What you can do

Users with this vulnerable versions should up data as fast as possible. And H2 Console should never be made available to untrusted users.

## What we are doing

> We are scanning the internet for vulnerable GitLab servers, and will notify system owners via the listed abuse contacts.

Reviewer: ```suggestion
We are scanning the internet for vulnerable H2 consoles, and will notify system owners via the listed abuse contacts.
```

{% include timeline.html %}

## More information

> * [CVE-2022-23221](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23221)

Reviewer: ```suggestion
* {% cve CVE-2022-23221 %}
```
Is korter.

* [H2 Database Console Remote Code Execution](https://packetstormsecurity.com/files/165676/H2-Database-Console-Remote-Code-Execution.html)

> * [CVE-2021-42392](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-42392)

Reviewer: ```suggestion
* {% cve CVE-2021-42392 %}
```
